core/src/main/resources/i18n/displayStrings.properties

```
@@ -480,6 +480,11 @@ createOffer.triggerPrice.tooltip=As protection against drastic price movements y
 createOffer.triggerPrice.invalid.tooLow=Value must be higher than {0}
 createOffer.triggerPrice.invalid.tooHigh=Value must be lower than {0}
 
+createOffer.buyBsq.popupMessage=Trading fees are paid to fund the Bisq DAO when making or taking an offer. Fees can be paid in BSQ or BTC.\n\n\
+  Bisq encourages traders to use BSQ by offering a discount of approximately 50%, since BSQ fees help fund Bisq's development. \
+  This discount varies as the BSQ/BTC rate fluctuates. To maintain the 50% discount target, trading fees are updated every cycle as necessary.\n\n\
+  For more details see [HYPERLINK:https://bisq.wiki/Trading_fees]
```

```suggestion
createOffer.buyBsq.popupMessage=Trading fees are paid to fund the Bisq DAO. Fees can be paid in BSQ or BTC.\n\n\
  BSQ fees directly help fund Bisq's development, so Bisq encourages traders to use BSQ by offering a 50% discount on trading fees. \
  This discount varies as the BSQ/BTC rate fluctuates. To maintain the 50% discount target, trading fees are updated every cycle as necessary.\n\n\
  For more about fees, see [HYPERLINK:https://bisq.wiki/Trading_fees]. For more about the Bisq DAO, see [HYPERLINK:https://bisq.wiki/Introduction_to_the_DAO#The_Bisq_DAO].
```

Just made some wording changes and added a link to the DAO introduction on the wiki.